pkg/operator/controller/ingress/controller.go

```diff
@@ -1063,10 +1074,10 @@ func (r *reconciler) ensureIngressController(ci *operatorv1.IngressController, d
 
 	var lbService *corev1.Service
```

Comment:
I think this is unnecessary now since you have:
`haveLB, lbService, err := r.ensureLoadBalancerService(ci, deploymentRef, platformStatus)`
which re-declares `lbService`

Reply:
Duh! That is correct. Updated.